lib/pid/PID.h

```diff
@@ -125,6 +125,33 @@ class PID
      */
     void setTunings(float Kc, float tauI, float tauD);
 
+    /**
+     * Changes proportional parameter (Kc) only
+     * 
+     * Not suited for auto tuning mode
+     * 
+     * @param Kc - Proportional Tuning parameter
+     */
+    void setKc(float Kc);
```

Comment:
shouldn't these be kP, kI, kD?

Reply:
the member vars in the pid class are named like this. tought it might be a good idea to keep it the same. do you think i should change them?

Reply:
@tandronescu @jetkov not sure what those variables stand for. are they analogous to kp, ki, kd?

Reply:
for reference the following is from the PID header file:
//Actual tuning parameters used in PID calculation.
    float Kc_;
    float tauR_;
    float tauD_;

Reply:
oh shoot no, they are not the same: https://en.wikipedia.org/wiki/PID_controller
under mathematical form